packages/@ourworldindata/grapher/src/tooltip/Tooltip.tsx

```diff
@@ -150,27 +154,17 @@ class TooltipCard extends React.Component<
                         {title && <div className="title">{title}</div>}
                         {subtitle && (
                             <div className="subtitle">
-                                {timeNotice && (
-                                    <FontAwesomeIcon icon={faInfoCircle} />
-                                )}
+                                {timeNotice && TOOLTIP_ICON.notice}
                                 <span>{subtitle}</span>
                             </div>
                         )}
                     </div>
                 )}
                 {children && <div className="content">{children}</div>}
-                {(notice || prompt) && (
+                {footer && (
                     <div className="endmatter">
-                        {notice && (
-                            <>
-                                <FontAwesomeIcon icon={faInfoCircle} />
-                                <p className="time-notice">
-                                    Data not available for {notice}. Showing
-                                    closest available data point instead.
-                                </p>
-                            </>
-                        )}
-                        {prompt && <p className="prompt">{prompt}</p>}
```

Comment:
Not sure here, did you remove `prompt` by choice?

Reply:
Yeah, `prompt` was just there for the 'Click to show changes' text in the map tooltip, but Max thought we should drop it since it was added before the sparkline existed to let people access essentially the same plot